### ISSUE TITLE
make ipackage fails — mbgl.xcodeproj does not contain a target named everything

### ISSUE THREAD

**Issue by gcamp** [mapbox/mapbox-gl-native#4590]
**Platform: iOS**
**Mapbox SDK version: master branch**

### Steps to trigger behavior

 1. type `make ipackage` in the root of the project

### Expected behavior

Successful build

### Actual behavior

I have the following error : 
```
xcodebuild: error: The project './build/ios-all/gyp/mbgl.xcodeproj' does not contain a target named 'everything'.
make: *** [ipackage] Error 65
```

Other project (OSX and Android) are building fine, this is only happening with iOS. Opening the problematic .xproj shows no target name 'everything', but one named 'All'. Should https://github.com/mapbox/mapbox-gl-native/blob/master/platform/ios/scripts/package.sh#L96 be updated respectively?

**Comment by 1ec5** [comment]
The script has been referring to `everything` for quite a while. Not sure if anything else changed at the gyp level to break this.

**Comment by jfirebaugh** [comment]
Could be due to #4561.

If we're not testing `make ipackage` in CI, we should remove that target.

If we don't want to remove that target, we should be testing it in CI.

**Comment by 1ec5** [comment]
We do build `make ipackage` in nightly builds on Bitrise.

**Comment by jfirebaugh** [comment]
Fixed in #4607.
